pkg/api/customization/roletemplate/roletemplate.go

```diff
@@ -49,3 +51,21 @@ func (w Wrapper) Validator(request *types.APIContext, schema *types.Schema, data
 	}
 	return nil
 }
+
+func (w Wrapper) Formatter(apiContext *types.APIContext, resource *types.RawResource) {
+	roleTemplates, err := w.RoleTemplateLister.List("", labels.Everything())
+	if err != nil {
+		logrus.Warnf("Failed to list roletemplates. Error: %v", err)
```

Comment:
Add a context here like `[roletemplate formatter]` so this can be traced back if needed

Reply:
oo good idea